notifier/selfstate/checkable/checkabletypes.go

```diff
@@ -0,0 +1,84 @@
+package checkable
+
+import (
+	"fmt"
+	"github.com/moira-alert/moira"
+)
+
+const (
+	RedisDisconnectedErrorMessage  = "Redis disconnected"
+	FilterStateErrorMessage        = "Moira-Filter does not receive metrics"
+	CheckerStateErrorMessage       = "Moira-Checker does not check triggers"
+	RemoteCheckerStateErrorMessage = "Moira-Remote-Checker does not check remote triggers"
+)
+
+const templateMoreThan = "%s more than %ds. Send message."
+
+// Checkable is the interface for simplified events verification.
+type Checkable interface {
+	Check(int64) *moira.NotificationEvent
+	NeedTurnOffNotifier() bool
+	NeedToCheckOthers() bool
+}
+
+// baseDelay basic structure for Checkable structures
+type baseDelay struct {
+	log moira.Logger
+	db  moira.Database
```

Comment:
In my opinion logger and database should be a pointers, shouldn't them?

Reply:
it is interface